crates/router/src/connector/airwallex/transformers.rs

```diff
@@ -365,6 +388,17 @@ pub struct AirwallexPaymentsResponse {
     next_action: Option<AirwallexPaymentsNextAction>,
 }
 
+#[derive(Default, Debug, Clone, Serialize, Deserialize, PartialEq)]
```

Comment:
Remove `Serialize` from the struct.

Also if possible please cross-check all the other structs and subsequently remove all the redundant derives

Reply:
Done